.github/workflows/test-code.yml

```diff
@@ -0,0 +1,70 @@
+name: Test code
+
+on:
+  pull_request:
+    branches:
+      - master
+      - develop
+    paths:
+      - src/**
+      - .github/workflows/**
+      - "*.sln"
```

Comment:
🤔 Sln file is not in the src folder. In other repositories, it is inside the src folder. What option do we choose?

Reply:
I thought about it and decided it was more appropriate here because it references files in this folder and the `.github` folder and so on.  If not, then it should be in the src folder.

Reply:
The sln file should not have to traverse up a directory - e.g. `../.github/workflows/testcode.yml`

Reply:
I think that the sln file used to only reference the projects, at which point it was most appropriately in the src folder.  But then you/we started adding references to the documentation and other files.  So now it really should be moved in the other repos.

Reply:
ok